packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2566,13 +2576,15 @@ class StatefulElement extends ComponentElement {
   }
 }
 
-abstract class _ProxyElement extends ComponentElement {
-  _ProxyElement(_ProxyWidget widget) : super(widget) {
+/// Instantiation of [ProxyElement]s.
```

Comment:
for consistency this should be "Base class for instantiations of [ProxyWidget] subclasses."